src/item_group.cpp

```diff
@@ -265,10 +264,17 @@ void Item_modifier::modify(item &new_item) const
             new_item = cont;
         }
     }
+
     if (contents.get() != NULL) {
         Item_spawn_data::ItemList contentitems = contents->create(new_item.bday);
         new_item.contents.insert(new_item.contents.end(), contentitems.begin(), contentitems.end());
     }
+
+    if( !custom_flags.empty() ) {
```

Comment:
This check is pointless. The following loop will simply do nothing if `custom_flags` is empty.